src/backends/onnx/onnx_backend.cc

```diff
@@ -75,17 +75,18 @@ OnnxBackend::Init(const std::string& path, const ModelConfig& config)
 
 Status
 OnnxBackend::CreateExecutionContexts(
-    const std::unordered_map<std::string, std::string>& paths)
+    const std::unordered_map<std::string, std::string>& models)
 {
   // [TODO] configurable like optimization policy in Tensorflow models
   // Create a "prototype" session option, which will be cloned and set
   // context-specific option on context creation.
-  OrtSessionOptions* session_options = OrtCreateSessionOptions();
-  OrtSetSessionThreadPoolSize(session_options, 1);
+  OrtSessionOptions* session_options;
+  RETURN_IF_ORT_ERROR(OrtCreateSessionOptions(&session_options));
+  RETURN_IF_ORT_ERROR(OrtSetSessionThreadPoolSize(session_options, 1));
   // disable graph optimization
-  OrtSetSessionGraphOptimizationLevel(session_options, 0);
+  RETURN_IF_ORT_ERROR(OrtSetSessionGraphOptimizationLevel(session_options, 0));
```

Comment:
Need to be careful on making sure the Ort resource is released, we have to call its release function explicitly to do so.

This is going to be tedious, you can probably wrap the Ort resource inside a class and let the destructor calls the release function for you when the wrapped class object goes out of scope.